src/test/isolation2/expected/vacuum_progress_column.out

```diff
@@ -341,12 +341,31 @@ select relid::regclass as relname, phase, heap_blks_total, heap_blks_scanned, he
  Success:        
  Success:        
 (3 rows)
+
+-- we shouldn't proceed until FTS probe has bumped the version
+2: SELECT gp_inject_fault('vacuum_rel_finished_one_relation', 'suspend', '', '', 'vacuum_progress_ao_column', 1, 1, 0, 1) FROM master();
+ gp_inject_fault 
+-----------------
+ Success:        
+(1 row)
 -- resume walsender and let it exit so that mirror stop can be detected
 2: SELECT gp_inject_fault_infinite('wal_sender_loop', 'reset', dbid) FROM gp_segment_configuration WHERE role = 'p' and content = 1;
  gp_inject_fault_infinite 
 --------------------------
  Success:                 
 (1 row)
+-- wait for the mirror stop to be detected (timeout 2 minutes)
+2: SELECT wait_for_mirror_down(1::smallint, 120);
```

Comment:
wait_for_mirror_down() initiates extra FTS probe loop between nodes, may be not one, while injected fault used previously used existing notifications, making less disturbance to the system under test, by the cost of extra code change. I am not sure which one is better.

Reply:
I think in this case extra FTS probes should be acceptable since all that part of the test is checking is that the backends will eventually be restarted. That way multiple FTS probes shouldn't affect the behavior since they don't bump status_version